src/fundus/publishers/es/__init__.py

```diff
@@ -0,0 +1,14 @@
+from fundus.publishers.base_objects import Publisher, PublisherGroup
+from fundus.publishers.es.el_pais import ElPaisParser
+from fundus.scraping.url import RSSFeed
+
+
+class ES(metaclass=PublisherGroup):
+    ElPais = Publisher(
+        name="El País",
+        domain="https://elpais.com/",
+        parser=ElPaisParser,
+        sources=[
+            RSSFeed("https://feeds.elpais.com/mrss-s/pages/ep/site/elpais.com/portada"),
```

Comment:
It seems like you can request sitemaps with this schema `https://elpais.com/sitemaps/2015/10/sitemap_0.xml`. Everything greater than `0` seems to result in the same sitemap.

Reply:
Nice, how did you find that?

Reply:
Often it helps googeling the site + sitemap, in this case `www.elpais.com sitemap`